backend/db/queries.py

```diff
@@ -2,11 +2,14 @@
 from functools import cache
 from typing import List, Dict
 from fastapi import Query
+from sqlalchemy import Connection
+
 from backend.db.utils import sql_query, sql_query_single_col, get_db_connection, sql_in
 
 
-def get_concepts(concept_ids: List[int], con=get_db_connection(), table:str='concepts_with_counts') -> List:
+def get_concepts(concept_ids: List[int], con: Connection = None, table:str='concepts_with_counts') -> List:
     """Get information about concept sets the user has selected"""
+    con = con if con else get_db_connection()
```

Comment:
I just made this same page in a number of functions:
Delete: `con=get_db_connection()`
Add: ` con: Connection = None` and ` con = con if con else get_db_connection()`